recipes/consensusfixer/ConsensusFixer.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash`
Prefer portable shebangs: https://blogs.gentoo.org/mgorny/2016/02/08/a-quick-note-on-portable-shebangs/

Reply:
Yup, good idea. I was just reusing @sposadac 's work on InDelFixer, but this makes it a bit more portable.
(We did use the same approach in SIB Vital-IT's RPMs).